src/App.tsx

```diff
@@ -15,47 +15,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType;
+  isReversed: boolean;
+};
+
+// Use this function in the render method to prepare goods
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  // To avoid the original array mutation
+  let visibleGoods = [...goods];
+
+  // Sort and reverse goods if needed
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods
+      = visibleGoods.sort((good1, good2) => good1.localeCompare(good2));
```

Comment:
lets use better naming and simplify it
```suggestion
    case SortType.ALPHABET:
     return firstGood.localeCompare(secondGood));
```